### PR TITLE
doc: Chinese docs update

### PR DESCRIPTION
**Pull request by VoVAllen**

### CODE REVIEW

**Comment by gaocegege** [comment]
Personally, I prefer keeping the intro and getting started on the same page. The analytics shows that most users only view the landing page. Thus we should keep essential things on the landing page (getting started in our case.)

File: docs/.vitepress/config/sidebar/main.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ export const mainSidebar: DefaultTheme.Sidebar = {
       text: 'Guide',
       collapsible: true,
       items:[
+        { text: "What's envd?", link: '/guide/intro' },
         { text: 'Getting Started', link: '/guide/getting-started' },
```
Comment (kemingy) [inline docs/.vitepress/config/sidebar/main.ts:10]: Click "Get Started" on the landing page will go to the 2nd page, which is a little weird.

**Comment by VoVAllen** [comment]
> Personally, I prefer keeping the intro and getting started on the same page. The analytics shows that most users only view the landing page. Thus we should keep essential things on the landing page (getting started in our case.)

SGTM.

**Comment by gaocegege** [comment]
LGTM

File: docs-zh/faq/comparison.md [hunk 2]
```diff
@@ -1,32 +1,30 @@
...
-- Cache support for standard installation tools, such as pip, conda, apt, and so on. In a standard dockerfile build, each layer depends on the previous layer. Any change that happens before will trigger the re-build for all the later stages, which greatly increases the build time. `envd` had built-in cache support for most common-used tools and reduced the dependency between them, which minimizes the time to wait when you make modifications to your environment. 
-- Python-like language syntax. `envd` use a python dialect called starlark as the language and provide a set of built-in functions to simplify your burden. Declare what you want, and `envd` will take care of the rest, including user permission, ssh server, entrypoint setup, etc...
+##它是如何工作的？
+- 并行镜像构建过程：在buildkit的支持下，`envd`可以将不同的工具安装并行化，以减少构建时间。
+- 复用常见包管理工具的缓存，如pip、conda、apt等。在一个标准的dockerfile构建中，每一层都依赖于前一层。之前发生的任何变化都会触发后面所有阶段的重新构建，这大大增加了构建时间。`envd`对大多数常用的工具都有内置的缓存支持，并减少了它们之间的依赖性，这使你在对环境进行修改时，再次构建的时间降到最低。
+- 类似Python的语言语法。`envd`使用一种叫做starlark的python方言作为语言，并提供了一组内置函数来简化你的负担。声明你想要的东西，`envd`会处理其余的事情，包括用户权限、ssh服务器、入口设置等……
```
Comment (kemingy) [inline docs-zh/faq/comparison.md:18]:
```suggestion
- 类似Python的语法。`envd`使用一种叫做starlark的python方言作为语言，并提供了一组内置函数来简化你的负担。声明你想要的东西，`envd`会处理其余的事情，包括用户权限、ssh服务器、入口设置等……
```

File: docs-zh/faq/comparison.md [hunk 3]
```diff
@@ -1,32 +1,30 @@
...
 
-## `envd` Benefits
-Benefits of using `envd` include:
-- Clear dev environment setup and management for each project. No longer worry about the conflict between CUDA, system package, virtualenv, etc. Each project will have its dev environment. And you can easily use the environment on any machine and share it with your collaborators. 
-- Unify dev and production environments by `envd`'s multi-target supports.
-- Integrate with container ecosystem. `envd` can be used in any container, such as Docker, Kubernetes, etc.
-- Dataset support. `envd` can be used to download and preprocess datasets.
+## `envd`能带来的好处
+使用`envd`的好处包括：
+- 为每个项目提供清晰可追踪的开发环境设置和管理。不再担心CUDA、apt包、virtualenv等之间的冲突。每个项目都会有自己的开发环境。而且你可以很容易地将这个环境部署到任何机器上，并且可以轻松将它分享给你的同事们。
```
[the remark this answers is not in the document]
Comment (gaocegege) [reply docs-zh/faq/comparison.md:10]: SGTM

**Comment by VoVAllen** [comment]
Added a new tool `prettier` and two makefile command `make lintmd` and `make prettiermd`.

However it's not included in CI now, because lots of files need to be fixed. I'll do this in a separate PR.

File: docs-zh/faq/comparison.md [hunk 4]
```diff
@@ -1,32 +1,35 @@
-# `envd` vs. Other tools
+# `envd` 和其他工具的比较
 
-## What is `envd`?
+## `envd`是什么?
 
-`envd` is a tool to build portable environments for machine learning and data science scenarios with Python-like syntax. 
+`envd`是一个为机器学习和数据科学场景建立可迁移，可复现环境的工具。
```
Comment (gaocegege) [inline docs-zh/faq/comparison.md:5]: Please add the blank between English and Chinese chars.

Comment (VoVAllen) [reply docs-zh/faq/comparison.md:5]: fixed

File: docs-zh/guide/getting-started.md [hunk 5]
```diff
@@ -1,157 +1,173 @@
...
-<td>
+````
+</details>
 
-```diff
-$ docker build
-=> pip install tensorflow      278s
-- => Downloading tensorflow-...-.whl (511.7 MB)
-```
+⏱️ **Builtkit原生，构建速度提高6倍**。
```
Comment (gaocegege) [inline docs-zh/guide/getting-started.md:72]:
```suggestion
⏱️ **Builtkit 原生，构建速度提高 6 倍**。
```

File: docs-zh/guide/getting-started.md [hunk 6]
```diff
@@ -1,157 +1,173 @@
...
 
-</td>
-</tr>
-</table>
+[Buildkit](https://github.com/moby/buildkit)支持并行构建和构建时的软件缓存（例如pip和apt缓存）。你可以在不用了解细节的情况下享受到它的强大之处。
```
Comment (gaocegege) [inline docs-zh/guide/getting-started.md:74]:
```suggestion
[Buildkit](https://github.com/moby/buildkit) 支持并行构建和构建时的软件缓存（例如 pip 和 apt 缓存）。你可以在不用了解细节的情况下享受到它的强大之处。
```

**Review by gaocegege — COMMENTED** [review]
Please review the blanks for all markdown flies.

**Comment by VoVAllen** [comment]
Did some research on `prettier` with markdown. It cannot handle HTML blocks in the markdown file properly. Therefore md file with HTML such as `<detail></detail>` cannot be formatted properly. 

And it won't format inline code block such like `build.envd`

Otherwise it can check the space between English and Chinese characters.

File: docs-zh/guide/getting-started.md (hunk 6 above)
Comment (VoVAllen) [reply docs-zh/guide/getting-started.md]: fixed

**Comment by gaocegege** [comment]
> Did some research on `prettier` with markdown. It cannot handle HTML blocks in the markdown file properly. Therefore md file with HTML such as `<detail></detail>` cannot be formatted properly.
> 
> And it won't format inline code block such like `build.envd`
> 
> Otherwise it can check the space between English and Chinese characters.

Awesome. Could we add it in github action?

**Review by gaocegege — APPROVED** [review]
/lgtm

**Pull request merged**